tutorials/foreign-object.html

```diff
@@ -0,0 +1,813 @@
+                        backgroundColor: '#131e29',
+                        height: '100%'
+                    },
+                    children: [
+                        {
+                            tagName: 'p',
+                            selector: 'text',
+                            style: {
+                                color: '#F2F2F2',
+                                font: '16px sans-serif',
+                                padding: '10px',
+                                margin: 0,
+                            },
+                            textContent: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.'
+                        }
+                    ]
+                }
+            ]
+        }
+    ]
+});
+
+const card = new Card({
+    position: { x: 10, y: 10 },
+    attrs: {
+        body: { width: 180, height: 100 }
+    }
+});
+
+card.addTo(graph);
+</code></pre>
+
+        <p>
+            As mentioned earlier, we would like to create an equivalent example with JointJS, that means selecting text needs
+            to be possible. In order to achieve this functionality, we need to use the <code>paper</code>
+            <a href="/docs/jointjs#dia.Paper.prototype.options.guard" target="_blank"><code>guard</code></a> option available
+            in JointJS to prevent the <code>paper</code> from handling UI events related to <code>&lt;p&gt;</code>
+            tags. In our <code>guard</code> function, we can return <code>true</code> if the <code>evt.target.tagName</code> 
+            is <code>'P'</code>.
+        </p>
+
+        <p>
+            While <code>guard</code> will handle events, we also need to allow the user select text via CSS. As the default CSS 
+            in JointJS sets the <code>user-select</code> property to <code>'none'</code>, we need to override this on our element.
+            The <code>guard</code> option combined the with following CSS override enables users to select text in the same manner 
+            as the vanilla SVG example from earlier. We can also adjust the <code>cursor</code> value in order to see the text cursor.
+        </p>
+
+        <pre><code>g[data-type="example.ForeignObject"] {
+    -webkit-user-select: text;  /* prefix needed for Safari */
+    user-select: text;
+}
+
+g[data-type="example.ForeignObject"] p {
+    cursor: auto;
+}
+</code></pre>
+
+        <p>
+            As our <code>foreignObject</code> might contain lots of HTML, there are moments when we certainly want to
+            be more concise when creating custom elements. Taking advantage of
+            <a href="/docs/jointjs#util.svg" target="_blank"><code>util.svg</code></a> in JointJS,
+            we can convert a string into a <a href="/docs/jointjs#dia.Cell.markup" target="_blank""><code>markup</code></a>
+            object while declaring a <code>Cell</code>. The resulting custom element will be more compact, and maybe easier to
+            read for some users.
```

Comment:
i would leave out the part which says "there are moments where we want to be more concise". Also, I would rework the last sentence. For example:

"Since our `foreignObject` contains quite a lot of HTML, we can take advantage of the `util.svg` method to make our `markup` object more concise. The resulting custom element definition will be more compact, and might prove easier to read for some developers."